src/bank02_player_menus.asm

```diff
@@ -6884,16 +6916,19 @@ call_02_75f4:
     ld   H, A                                          ;; 02:760d $67
     ld   A, [wMoneyLow]                                ;; 02:760e $fa $be $d7
     ld   L, A                                          ;; 02:7611 $6f
-    ld   A, H                                          ;; 02:7612 $7c
-    or   A, L                                          ;; 02:7613 $b5
-    jr   NZ, .jr_02_761c                               ;; 02:7614 $20 $06
-    add  A, $30                                        ;; 02:7616 $c6 $30
-    call storeTileAatDialogPositionDE                  ;; 02:7618 $cd $44 $38
-    xor  A, A                                          ;; 02:761b $af
 .jr_02_761c:
     push DE                                            ;; 02:761c $d5
     push BC                                            ;; 02:761d $c5
-    call drawNumberAtDialogPositionDE                  ;; 02:761e $cd $18 $5b
+    ld   C, $00
+    ld   A, [wStatusEffect]
+    bit  7, A
```

Comment:
This could be replaced with `rla` to save a byte.

Reply:
Done